### ISSUE TITLE
Support For Article mode in Firefox Based Browsers

### ISSUE THREAD

**Issue by JYamihud** [lbryio/lbry-desktop#6441]
There is a little icon in some web pages on GNU IceCat ( and other Firefox based browsers ) 

![image](https://user-images.githubusercontent.com/27649397/125230063-b8729200-e2e0-11eb-843e-0672503f59f4.png)

It looks like this. And what it does. It turns an article that looks like this:
![image](https://user-images.githubusercontent.com/27649397/125230115-da6c1480-e2e0-11eb-84b8-cbf4627800f4.png)

Into something that's easier to read. Like this:
![image](https://user-images.githubusercontent.com/27649397/125230170-fb346a00-e2e0-11eb-8737-f60fe6f3c4ac.png)

Basically, it removes all the unnecessary parts of the page, make you focus only on the words you are reading. And I love this feature very much. 

![image](https://user-images.githubusercontent.com/27649397/125230297-35057080-e2e1-11eb-86f2-8f61f5c71587.png)

This is an article on Odysee. Where is the icon? It fails to recognise the text in the article. Please make it work.

**Comment by btzr-io** [comment]
The problem is the loading time.  Dynamic content is not detected after change. Not sure if there is anything that can be done to fix this.

**Comment by JYamihud** [comment]
Maybe we cane make a simple IF statement. 
IF publication type is article: Put the text of the article, right there in the HTML of the page.

I mean, text is quite quick anyway. It doesn't need dynamic loading.

**Comment by btzr-io** [comment]
> I mean, text is quite quick anyway. It doesn't need dynamic loading.

You can't just make the text appear, it needs to load from the streaming url.

**Comment by JYamihud** [comment]
Can it be done on the server side, before it loads the page? Or can it force reload something to update the HTML of the page?
